File: src/Filme.java
public class Filme extends Obra{
    private int qntMinutos;
    public String diretor;

    public Filme(String title, int qntMinutos, String diretor) {
        super(title);
        this.qntMinutos = qntMinutos;
        this.diretor = diretor;
    }
    public String tamanhoObra(){
        return "Duração do filme: " + this.qntMinutos + " minutos";
    };
    public String diretor(){
        return "Diretor: " + this.diretor;
    };
}

File: src/Filme.kt
class Filme(title: String?, private val qntMinutos: Int, var diretor: String) : Obra(title) {
    override fun tamanhoObra(): String {
        return "Duração do filme: " + qntMinutos + " minutos"
    }

    fun diretor(): String {
        return "Diretor: " + diretor
    }
}